rust/src/plutus.rs

```diff
@@ -52,36 +52,60 @@ impl PlutusScripts {
 #[wasm_bindgen]
 #[derive(Clone, Debug, Eq, Ord, PartialEq, PartialOrd)]
 pub struct ConstrPlutusData {
-    tag: Int,
+    alternative: BigNum,
```

Comment:
from what i can tell this will be a breaking change for any dapps currently using `tag` then